devices/patches/rtc/h7_hi_mem.yaml

```diff
@@ -0,0 +1,3 @@
+_modify:
+  ICSR:
+    name: ISR
```

Comment:
@burrbull 
What am I doing wrong here?

Without this fix:
```
$ cargo c --features stm32h7b3,rt,defmt
[...]
error[E0599]: no method named `isr` found for struct `Periph<stm32h7::stm32h7b3::rtc::RegisterB
lock, 1476411392>` in the current scope
   --> src/rtc.rs:765:26
    |
765 |                 self.reg.isr().modify(|_, w| w.wutf().clear_bit());
    |                          ^^^
    |
help: there is a method `icsr` with a similar name
    |
765 -                 self.reg.isr().modify(|_, w| w.wutf().clear_bit());
765 +                 self.reg.icsr().modify(|_, w| w.wutf().clear_bit());
[...]
```

Reply:
For reference I can see [ICSR here](https://stm32-rs.github.io/stm32-rs/STM32H7A3.html#RTC:ICSR)

Reply:
Or perhaps this should not even be done in the first place. There does seem to be quite a bit of differences between the RTC in RM0455 as compared to RM0433. In RM0433 we have the battery backed ram in RTC_BKP[0-31]R but I can not seem to find that in RM0455 even though it is mentioned as a feature from what I can tell.

Reply:
You should not do this on PAC level. H7A/B have newer RTC.
ISR and ICSR work different.